pyspectral/utils.py

```diff
@@ -433,7 +433,7 @@ def download_luts(**kwargs):
     else:
         aerosol_types = HTTPS_RAYLEIGH_LUTS.keys()
 
-    chunk_size = 10124
+    chunk_size = 4096
```

Comment:
According to https://requests.readthedocs.io/en/master/api/#requests.Response.iter_content the chunk size is number of bytes so this is only 4K. I think we want something in the range of 1-10MB.

Reply:
I just fixed that, you are too quick @djhoese ! :-)

Reply:
We are downloading files in the order of 3-75 MB